modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -15,14 +15,8 @@ partition-size=0x1e000
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_config"
 
 partition=MCUBOOT_PAD
-if SOC_SERIES_NRF54LX
-	partition-size=0x800
+partition-size=0x200
```

Comment:
I do not think this change is ok. Look at this comment: https://github.com/nrfconnect/sdk-nrf/pull/14246#discussion_r1505590639
there is minimal size for alignment, that depends on IRQs reserved. This platform reserves 271 which in turn gives more than 2k for VTOR table, and when aligned to next power of two gives 4k (0x800).

Are you sure about this change?

Reply:
You are both right. I removed the commit to alter the mcuboot padding size and I fixed my alignment requirements by modifying the TF-M partitions themselves. 

FYI: @nvlsianpu